src/main/java/at/hannibal2/skyhanni/features/event/diana/InquisitorWaypointShare.kt

```diff
@@ -26,16 +26,29 @@ import net.minecraft.network.play.server.S02PacketChat
 import net.minecraftforge.event.entity.EntityJoinWorldEvent
 import net.minecraftforge.fml.common.eventhandler.EventPriority
 import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import java.util.regex.Matcher
 import kotlin.time.Duration.Companion.seconds
 
 object InquisitorWaypointShare {
 
     private val config get() = SkyHanniMod.feature.event.diana.inquisitorSharing
 
     private val patternGroup = RepoPattern.group("diana.waypoints")
+    /**
+     * REGEX-TEST: §9Party §8> User Name§f: §rx: 2.3, y: 4.5, z: 6.7
+     */
     private val partyPattern by patternGroup.pattern(
         "party",
-        "§9Party §8> (?<playerName>.*)§f: §rx: (?<x>-?[0-9]{1,4}), y: (?<y>-?[0-9]{1,4}), z: (?<z>-?[0-9]{1,4})\\b"
+        "§9Party §8> (?<playerName>.+)§f: §rx: (?<x>[^ ]+),? y: (?<y>[^ ]+),? z: (?<z>[^ ]+)"
+    )
+
+    //Support for https://www.chattriggers.com/modules/v/inquisitorchecker
+    /**
+     * REGEX-TEST: §9Party §8> UserName§f: §rA MINOS INQUISITOR has spawned near [Foraging Island ] at Coords 1 2 3
+     */
+    private val inquisitorCheckerPattern by patternGroup.pattern(
+        "party-inquisitorchecker",
```

Comment:
```suggestion
        "party.inquisitorchecker",
```
fix not matching shape